binancial/__init__.py

```
@@ -0,0 +1,9 @@
+from .backtester import BackTester
```

Ruff (0.8.2)

`.backtester.BackTester` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)